.travis.yml

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2013-2018 Arm Limited. All rights reserved.
```

Comment:
Please fix up the commit message.

Reply:
@jainvikas8 what's missing from the commit message?